src/shogun/machine/IterativeMachine.h

```diff
@@ -112,7 +115,9 @@ namespace shogun
 		virtual void end_training()
 		{
 		}
-
+		
+		/** Stores features to continue training */
+		CDotFeatures* m_continue_features;
```

Comment:
I think this would need to be CFeatures*, or? Why would we restrict to CDotFeatures?
Sorry it has been a while, we might have discussed this before ;)

Reply:
It should be CFeatures* but then we will need to check feature type in each iteration... for now we are only using IterativeMachine in LinearMachines which are DotFeatures.

Reply:
I think we could get away with a static cast and rely on the fact that the algorithm *knows* the feature type is correct (inside `iteration`), since it was checked before the algorithm started (by your last piece of gsoc work)